### PR TITLE
examples/hello: update Makefile

### PR DESCRIPTION
**Pull request by drvink**
Fix OCaml testseq and F# hello

### CODE REVIEW

**Comment by s-zanella** [comment]
Thanks! There are other problems with this Makefile that need to be fixed. For instance, there are other targets still using `--use_native_int` and references to `$(ULIB_ML)/native_int` (now gone).

**Comment by drvink** [comment]
I tried for a while to get `bat_testseq` to build but couldn't, so I left its target alone. I pushed a new commit to remove the other stale bits just now.

File: examples/hello/Makefile [hunk 1]
```diff
@@ -15,9 +15,9 @@ ocaml: out hello.fst
 # In this particular case, we need to compile against the extracted version of
 # `FStar.Seq` (it is not realized in ML), so we pass it to `ocamlopt`.
 testseq: out testseq.fst
-	$(MAKE) -C $(ULIB_ML) PRIMS_DIR=native_int
-	$(FSTAR) $(FSTAR_DEFAULT_ARGS) --use_native_int --odir out --codegen OCaml testseq.fst
-	$(OCAMLOPT) -I $(ULIB_ML)/native_int -I out/ out/FStar_Seq.ml out/TestSeq.ml -o testseq.exe
+	$(MAKE) -C $(ULIB_ML) PRIMS_DIR=.
```
Comment (msprotz) [inline examples/hello/Makefile]: you should be able to just remove PRIMS_DIR=.

File: examples/hello/Makefile [hunk 2]
```diff
@@ -41,7 +41,7 @@ FS_LIBS=$(BIN)/FSharp.PowerPack.dll
 fs: out hello.fst
 	$(FSTAR)   --odir out --codegen FSharp hello.fst
 	cp $(FS_LIBS) out
-	$(FSC) --nowarn:0086 -o out/hello.exe $(LIB)/fs/prims.fs $(LIB)/fs/io.fs out/Hello.fs
+	$(FSC) --nowarn:0086 -o out/hello.exe ../../ulib/fs/prims.fs ../../ulib/fs/io.fs out/Hello.fs
```
Comment (msprotz) [inline examples/hello/Makefile]: why not change the definition of LIB above?

**Comment by beurdouche** [comment]
Fixed by @catalin-hritcu in 87f76389058853bf9cddf78c5b6fde8b107b6f7a

**Pull request closed without merging**